fdbserver/workloads/ConsistencyCheck.actor.cpp

```diff
@@ -1983,7 +2062,21 @@ struct ConsistencyCheckWorkload : TestWorkload {
 		    }
 		}*/
 
-		self->bytesReadInPreviousRound = bytesReadInthisRound;
+		if (consistencyCheckEpoch == 0) {
```

Comment:
Why do we print this when `consistencyCheckEpoch == 0`? Shall we do that only when task is complete `without` failedRanges?

Reply:
consistencyCheckEpoch == 0 will wait until all its children checkers complete. So, at this point, all children complete, and the job completes.